{"name": "test_sign_up_invalid_phone_num3", "status": "passed", "description": "the test is verify, that user can't sign up with invalid phone number", "start": 1680348741337, "stop": 1680348742056, "uuid": "347aff0a-d936-4b82-8d72-c8f9f0c234b9", "historyId": "80051cee47059ac718aa3cfe221dd2de", "testCaseId": "98cdbe3a2daee48437d0250a2dccc868", "fullName": "tests.chromeTests.testSignUpModal.TestSignUpModal#test_sign_up_invalid_phone_num3", "labels": [{"name": "parentSuite", "value": "tests.chromeTests"}, {"name": "suite", "value": "testSignUpModal"}, {"name": "subSuite", "value": "TestSignUpModal"}, {"name": "host", "value": "krot"}, {"name": "thread", "value": "12616-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.chromeTests.testSignUpModal"}]}